docs/external-postgres.md

```diff
@@ -0,0 +1,59 @@
+# Using an external database with the Chronicle Helm chart
+
+Chronicle uses a PostgreSQL database to store data. By default, Chronicle will
+use an embedded PostgreSQL database. However, you can configure Chronicle to
+use an external PostgreSQL database instead.
+
+## Configure Chronicle to use an external PostgreSQL database
+
+The default values.yaml for the Chronicle helm chart configures an
+internal Postgres database.
+To use an external database, you must override these values in your
+own values.yaml file.
+
+NOTE: The database must exist, Chronicle will not create it for you.
+
+The following example values.yaml file configures Chronicle to use
+an external PostgreSQL database.
+
+```yaml
+postgres:
+  enabled: false
+  host: my-postgres-host
+  port: 5432
+  database: my-postgres-database
+  user: my-postgres-username
+  password: my-postgres-password
+```
+
+Rather than setting the password in the values.yaml file, you can use a
+Kubernetes secret. For example:
```

Comment:
```suggestion
Rather than setting the password in the values.yaml file, it is
recommended that you use a Kubernetes secret. For example:
```

Reply:
Very good recomenataion, I've added it and reworded that section a little